src/index.html

```diff
@@ -7,8 +7,129 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+    rel="preconnect" 
+    href="https://fonts.gstatic.com" 
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a> 
+      <nav>
+        <ul class="list">
+          <li>
+            <a href="/" class="is-active listName">
+              Apple
+            </a>
+          </li>
+          <li>
+            <a href="/" class="listName">Samsung</a>
```

Comment:
```suggestion
            <a href="#" class="listName">Samsung</a>
```
https://www.loom.com/share/093826f859b646caac2c27cd4a5c2809
виправ, будьласка, щоб не було помилки 404